### PR TITLE
Outbreak input, chips fixed and input type changed

### PR DESCRIPTION
**Pull request by alaapbharadwaj**
## Description

Fixes two bugs with the outbreak name input. One changes the input type to a multi select drop down, and the other makes sure the chips show up when a result is searched

## Tickets

* [CNFT1-2808](https://cdc-nbs.atlassian.net/browse/CNFT1-2808)
* [CNFT1-2843](https://cdc-nbs.atlassian.net/browse/CNFT1-2843)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2808]: https://cdc-nbs.atlassian.net/browse/CNFT1-2808?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-2843]: https://cdc-nbs.atlassian.net/browse/CNFT1-2843?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx [hunk 1]
```diff
@@ -7,11 +7,28 @@ import {
...
+    const [outbreakNames, setOutbreakNames] = useState<Selectable[]>();
+
+    useEffect(() => {
+        ConceptOptionsService.conceptSearch({
+            name: outbreakCodeSetNm,
+            criteria: ''
+        }).then((response) => {
+            setOutbreakNames(response.options);
+        });
+    }, [outbreakCodeSetNm]);
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx:30]: Question: `useOptions`?

File: apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx [hunk 2]
```diff
@@ -7,11 +7,28 @@ import {
...
+import { ConceptOptionsService } from 'generated';
+import { useEffect, useState } from 'react';
+import { Selectable } from 'options';
 
-const CriteriaSearchFields = () => {
+type Props = {
+    outbreakCodeSetNm?: string;
+};
+
+const CriteriaSearchFields = ({ outbreakCodeSetNm = 'OUTBREAK_NM' }: Props) => {
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx:18]: Question: Do we need to pass in this as a prop? When will it be a different value?

File: apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx (hunk 1 above)
Comment (adamloup-enquizit) [reply apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx:30]: This use case is prevalent enough for a `ConceptMultiSelect` component to be created that takes the code set name as a parameter.

File: apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx (hunk 2 above)
Comment (alaapbharadwaj) [reply apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx]: Just changed after adams suggestion of creating a lone component to handle the concept multi selects that could be used elsewhere as well

**Review by adamloup-enquizit — APPROVED** [review]
Looks great!  That new component will help standardize value set usage 👍

**Review by stevegsa — APPROVED** [review]

**Pull request merged**